### PR TITLE
[DO-1989] Add preloader

### PR DESCRIPTION
**Pull request by Iva-khar**

### CODE REVIEW

File: src/index.jsx [hunk 1]
```diff
@@ -11,6 +11,10 @@ import 'moment/locale/uk';
 import './i18n';
 
 
+$(window).on('load', () => {
+  $('#preloader').fadeOut('slow');
+});
```
Comment (roman-tiukh) [inline src/index.jsx:16]: Nope, you have `React` and init state

File: public/index.html [hunk 2]
```diff
@@ -15,6 +15,8 @@
     <title>Data Ocean</title>
   </head>
   <body>
+    <div id="preloader"></div>
```
Comment (roman-tiukh) [inline public/index.html:18]: Preloader should be controlled by `React`

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: src/index.jsx (hunk 1 above)
Comment (Iva-khar) [reply src/index.jsx]: rewrited

File: public/index.html (hunk 2 above)
Comment (Iva-khar) [reply public/index.html:18]: rewrited

**Pull request closed without merging**